### PR TITLE
docs: add details for contrib summit NA social

### PR DESCRIPTION
**Pull request by salaxander**

### CODE REVIEW

File: content/en/events/2023/kcsna/social.md [hunk 1]
```diff
@@ -6,15 +6,20 @@ weight: 5
...
 
 ### Location
 
-TBD, check back soon!
+[Revel Motor Row](https://www.revelspace.com/our-spots/motor-row/)
+2400 S Michigan Ave
+Chicago, IL 60616
+
+### Poetry Slam
+This year the contributor summit social will host a Kubernetes themed poetry slam. To register ahead of the event, please add your name to [this list](https://docs.google.com/spreadsheets/d/1NiPRrT5HpM42O8DQTv41DRufFiVXNcO39BFm50Uj1Ac/edit#gid=0).
```
Comment (sftim) [inline content/en/events/2023/kcsna/social.md:22]:
```suggestion
This year the contributor summit social will host a Kubernetes themed poetry slam. To register ahead of the event, please add your name to [Kubernetes Contributor Summit Social Poetry Slam Registration](https://docs.google.com/spreadsheets/d/1NiPRrT5HpM42O8DQTv41DRufFiVXNcO39BFm50Uj1Ac/edit#gid=0).
```

**Review by sftim — COMMENTED** [review]
One nit

**Review by pnbrown — APPROVED** [review]

**Pull request merged**